source/docs/software/dashboards/shuffleboard/layouts-with-code/retrieving-data.rst

```diff
@@ -6,19 +6,39 @@ Unlike ``SmartDashboard.getNumber`` and friends, retrieving data from Shuffleboa
 
    .. code-block:: java
 
-       class DriveBase extends Subsystem {
-          private ShuffleboardTab tab = Shuffleboard.getTab("Drive");
-          private NetworkTableEntry maxSpeed =
-              tab.add("Max Speed", 1)
-                 .getEntry();
-
-          private DifferentialDrive robotDrive = ...;
-
-          public void drive(double left, double right) {
-            // Retrieve the maximum speed from the dashboard
-            double max = maxSpeed.getDouble(1.0);
-            robotDrive.tankDrive(left * max, right * max);
-          }
-       }
+      class DriveBase extends Subsystem {
+         private ShuffleboardTab tab = Shuffleboard.getTab("Drive");
+         private NetworkTableEntry maxSpeed =
+            tab.add("Max Speed", 1)
+               .getEntry();
+
+         private DifferentialDrive robotDrive = ...;
+
+         public void drive(double left, double right) {
+         // Retrieve the maximum speed from the dashboard
+         double max = maxSpeed.getDouble(1.0);
+         robotDrive.tankDrive(left * max, right * max);
+         }
+      }
+
+   .. code-block:: python
```

Comment:
did you intentionally not add C++?

Reply:
I didn't add C++ cause I always mess it up.  Wouldn't mind inserting it though.